PennCourses/settings/prod.py

```diff
@@ -24,6 +26,24 @@
 # Honor the 'X-Forwarded-Proto' header for request.is_secure()
 SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
 
+# Share cookie with frontend
+SESSION_COOKIE_DOMAIN = '.pennlabs.org'
```

Comment:
We don't need to do any cookie sharing for this or CSRF

Reply:
This is in anticipation of the accounts merge-in where we'll have a session cookie

Reply:
Right, but the cookie doesn't need to be shared. The idea is the frontend (in the user's browser) makes requests to api.penncourses.org and django will get access to the cookie directly. So the cookie doesn't need to be manually shared with those domains.